IBAnimatable/TransitionAnimationType.swift

```diff
@@ -23,6 +24,8 @@ public enum TransitionAnimationType {
   static func fromString(transitionType: String) -> TransitionAnimationType? {
     if transitionType.hasPrefix("Fade") {
       return fadeTransitionAnmationType(transitionType)
+    } else if transitionType.hasPrefix("SystemRippleEffect") {
```

Comment:
You are right, because `TransitionAnimationType` doesn't inherit from `String`, we can't use `init(rawValue: String)`. I haven't found any solutions for this. I am happy with your current implementation. Please merge it when you feel it is ready. 👍